.changeset/great-cherries-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+"@khanacademy/wonder-blocks-icon-button": minor
+"@khanacademy/wonder-blocks-button": minor
+"@khanacademy/wonder-blocks-clickable": patch
```

Comment:
**nit:** In this case, Clickable doesn't require a version bump as we are only modifying tests (not code that will be included in the bundle).

Reply:
would removing that line be enough to not generate the new patch? i saw that you can do `yarn changeset --empty` but i think that only works if it's just one package